x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -317,6 +306,38 @@
                 "target_field": "destination.geo",
                 "ignore_missing": true
             }
+        },
+        {
+            "remove": {
+                "field": [
+                    "suricata.eve.alert.action",
+                    "suricata.eve.alert.category",
+                    "suricata.eve.alert.severity",
+                    "suricata.eve.app_proto",
+                    "suricata.eve.dest_ip",
+                    "suricata.eve.dest_port",
+                    "suricata.eve.event_type",
+                    "suricata.eve.fileinfo.filename",
+                    "suricata.eve.fileinfo.size",
+                    "suricata.eve.flow.end",
+                    "suricata.eve.flow.start",
+                    "suricata.eve.flow.bytes_toclient",
+                    "suricata.eve.flow.bytes_toserver",
+                    "suricata.eve.flow.pkts_toclient",
+                    "suricata.eve.flow.pkts_toserver",
+                    "suricata.eve.http.hostname",
+                    "suricata.eve.http.http_method",
+                    "suricata.eve.http.http_refer",
+                    "suricata.eve.http.http_user_agent",
+                    "suricata.eve.http.length",
+                    "suricata.eve.http.status",
+                    "suricata.eve.proto",
+                    "suricata.eve.src_ip",
+                    "suricata.eve.src_port",
+                    "suricata.eve.timestamp"
+                ],
+                "ignore_missing": true
+            }
```

Comment:
For most of these fields, you could actually get rid of the `convert { type: string}` and do a straight field rename. Would look much more like the other modules, and would remove the need to have this huge `remove` operation.

Reply:
You are completely right. There's a lot of cleanup that can be done